Files/UserControls/StatusBarControl.xaml

```diff
@@ -47,6 +47,23 @@
             HorizontalAlignment="Right"
             Orientation="Horizontal"
             Spacing="2">
+            
+            <Button 
+                x:Name="PreviewPane"
+                x:Uid="PreviewPaneToggle"
+                FontFamily="{StaticResource FluentUIGlyphs}" Content="&#xEA8D;"
+                Width="36"
+                Height="32"
+                Margin="2,0,2,0"
```

Comment:
We made some changes to the spacing an margins in `master`, can you try synchronizing with `master` to make sure this is styled correctly? Do you have any thoughts on making this a toggle button?

Reply:
@yaichenbaum 
I'll update the margins now.
I didn't make this a toggle button because I couldn't get the custom styling used for the other buttons to work properly.
Instead of turning the accent color, it would just show the hover state when pressed.